modules/op-using-repository-crd-with-pipelines-as-code.adoc

```diff
@@ -0,0 +1,41 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="using-repository-crd-with-pipelines-as-code_{context}"]
+= Using the `Repository` CRD with {pac} 
+
+[role="_abstract"]
+The `Repository` CR has the following primary functions:
+
+* Inform {pac} about processing an event from a URL.
+* Inform {pac} about the namespace for the pipeline runs.
+* Reference an API secret, username, or an API URL necessary for Git provider platforms when using webhook methods. 
+* Provide the last pipeline run status for a repository.
+
+You can use the `tkn pac` CLI or other methods to create a `Repository` CR inside the target namespace. For example:
+
+[source,terminal]
+----
+cat <<EOF|kubectl create -n my-pipeline-ci -f- <1>
+
+apiVersion: "pipelinesascode.tekton.dev/v1alpha1"
+kind: Repository
+metadata:
+  name: project-repository
+spec:
+  url: "https://github.com/linda/project"
+EOF
+----
+<1> `my-pipeline-ci` is the target namespace.
+
+Whenever there is an event coming from the URL `https://github.com/linda/project`, {pac} matches it and starts checking out the content of the `linda/project` repository for pipeline run to match in the `.tekton/` directory.
```

Comment:
```suggestion
Whenever there is an event coming from the URL such as `https://github.com/linda/project`, {pac} matches it and starts checking out the content of the `linda/project` repository for pipeline run to match in the `.tekton/` directory.
```

**Question:** when you say "to match **in** the `.tekton/` directory.", do you mean the event or the content? 

**Suggestion:** Needs a rewrite for clarity.

Reply:
@@sm43 and @savitaashture Could you please help? Is it the event or the content?

Example:
Whenever there is an event coming from the URL such as `https://github.com/linda/project`, {pac} matches it and starts checking out the content of the `linda/project` repository for pipeline run to match the **content** in the `.tekton/` directory.

Reply:
Thank you!
Comment addressed!